package/tensorflowapimodel-faster-rcnn-resnet101-kitti/kitti_label_map.pbtxt

```diff
@@ -0,0 +1,9 @@
+item {
+  id: 1
+  name: 'car'
+}
+
+item {
+  id: 2
+  name: 'pedestrian'
+}
```

Comment:
Hmm, no 'cyclists'?

Reply:
This model don't know anything  about 'cyclists' - just cars and pedestrian